content/en/agent/basic_agent_usage/_index.md

```diff
@@ -150,7 +150,9 @@ When the Agent is running, use the `datadog-agent launch-gui` command to open th
 | [Windows 64-bit][10]                            | Windows 7+                                        |
 | [Windows Azure Stack HCI OS][10]                | All Versions                                      |
 
-**Note**: [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.
+**Note**: 
+- [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.
+- Windows Server 2008 R2+ is supported, but must have the most recent updates installed in order to run versions 7+ of the Datadog Agent. There is also [a known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.
```

Comment:
I think we would need the update to run the latest version of 6 too, no ?

Reply:
The issue referenced is specific to Python3, some Windows Updates have to be installed for it to work. Agent 6 is fine as long as they are using Python2.7.